.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @alltilla @smortex
```

Comment:
@alltilla are you okay to be added as a code owner?

This is mainly to get review requests assigned automatically.  My "standard policy" on this kind of repo is that I like to have a 3rd party approval before merging, and after a few days, if nobody complained, it's fine to merge changes.

Reply:
Yes, thank you.